Source/Libs/GtkSharp/Container.cs

```diff
@@ -51,6 +51,8 @@ struct GParamSpec
             public uint param_id;
         };
 
+        protected Container() : base() { }
```

Comment:
hy this change is needed?

the autogenerated code looked like this:

```
		protected Container() : base(IntPtr.Zero)
		{
			CreateNativeObject (new string [0], new GLib.Value [0]);
		}
```

Reply:
Don't know, taken from #90

Reply:
Reverted